packages/rhi-webgl/src/GLTexture2D.ts

```diff
@@ -91,7 +91,12 @@ export class GLTexture2D extends GLTexture implements IPlatformTexture2D {
     this._bind();
     gl.pixelStorei(gl.UNPACK_FLIP_Y_WEBGL, +flipY);
     gl.pixelStorei(gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, +premultiplyAlpha);
-    gl.texSubImage2D(this._target, mipLevel, x || 0, y || 0, baseFormat, dataType, imageSource);
+
+    if (this._useTexStorage) {
```

Comment:
The logic is not right, when format is `gl.LUMINANCE_ALPHA` and  `baseFormat === gl.ALPHA`